config/project-scratch-def.json

```diff
@@ -1,8 +1,8 @@
 {
   "orgName": "Special Authority Scratch Org",
   "edition": "Developer",
-  "description": "This org is created for Spring'23 org ",
-  "features": ["PersonAccounts", "LightningServiceConsole", "ServiceCloud","Communities","Sites","HealthCloudAddOn","ContactsToMultipleAccounts","OmniStudioDesigner","OmniStudioRuntime","Assessments"],
+  "description": "This org is created for Spring'23 org",
+  "features": ["PersonAccounts", "LightningServiceConsole", "ServiceCloud","Communities","Sites","HealthCloudForCmty","HealthCloudAddOn","ContactsToMultipleAccounts","OmniStudioDesigner","OmniStudioRuntime","InsuranceCalculationUser","Assessments"],
```

Comment:
@NataliaNikishina Flagging these changes for awareness for EDRD/SAT: "features": ["PersonAccounts", "LightningServiceConsole", "ServiceCloud","Communities","Sites","HealthCloudForCmty","HealthCloudAddOn","ContactsToMultipleAccounts","OmniStudioDesigner","OmniStudioRuntime","InsuranceCalculationUser","Assessments"]